src/adapters/native.js

```diff
@@ -0,0 +1,95 @@
+var mkFhir = require('../fhir');
+
+// Fetch Helper JSON Parsing
+function parseJSON(response) {
+
+  return response.text()
```

Comment:
Could you comment on why [response.json()](https://developer.mozilla.org/en-US/docs/Web/API/Body/json) isn'te used instead of this function?